spec/factories/case/sars.rb

```diff
@@ -211,6 +211,16 @@
     end
   end
 
+  factory :extended_deadline_sar, parent: :accepted_sar do
```

Comment:
The factories in `case/sars.rb` all seem to have separate factories for different states so continued that pattern.

Reply:
The other factories use traits and since the extension can happen in different states it makes sense to have it as a trait.
There is a pattern for this in the foi factories

Reply:
I'm leaving this as is as I followed the current implementation in factories/cases/sars.rb which has a factory per state.

Reply:
Exactly we have a factory per state. Extended sar is not a state so we should have a trait for it rather than a factory such as in factories/cases/fois.rb.